src/main/java/racingcar/presentation/ClientResponseConverter.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.presentation;
+
+import racingcar.presentation.dto.CarData;
+import racingcar.presentation.dto.GameResultResponse;
+import racingcar.domain.Car;
+import racingcar.domain.RacingGame;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ClientResponseConverter {
+
+    public static GameResultResponse toGameResultResponse(final RacingGame racingGame) {
+        return new GameResultResponse(
+                toWinnerResponse(racingGame),
+                toCarDataResponse(racingGame)
+        );
```

Comment:
```suggestion
        return new GameResultResponse(toWinnerResponse(racingGame), toCarDataResponse(racingGame));
```
100자를 안넘겨서 한줄로 해도 될것 같아요!